doc/smart-switch/PA-Validation/SmartSwitchPAValidationOffload.md

```diff
@@ -42,6 +42,9 @@ This document provides general information about offloading PA validation featur
 - Use existing ACL tables and ACL rules design in NPU.
 
 ## 1.2 Scale Requirements
+* 4096 PA validation entries
+
+As described in the [SONiC-DASH HLD](https://github.com/sonic-net/SONiC/blob/master/doc/dash/dash-sonic-hld.md#3213-underlay-src-ip-pa-validation)

```

Comment:
4K entries would be only for actual PA validation entries and there would be additional entries for catch-all and drop. Please include additional entries for {*, VNI}-->Drop.

Reply:
updated